### ISSUE TITLE
RStudio 1.1.383 issues with rsession and View(). Needed to use Task manager to force-quit

### ISSUE THREAD

**Issue by Courvoisier13** [rstudio/rstudio#1747]
Hi,
I am under Windows 7 64 bits (see below)
I have upgraded to RStudio 1.1.383 at work in the corporate env after trying it at home. I have had the same issues as everyone else at home (under Windows 10) with rstudio not doing anything and having to kill rsession.exe. I have solved the issue at home by deleting both `%localappdata%\RStudio-Desktop` & `%AppData%\RStudio` like in #1612 . but at work things are not working correctly. The issue comes back under the following conditions:

1. If I do not delete the `.Rproj.user` folder for the pre existing project => Rstudio gets stuck and hangs and can only close it by killing `rsession.exe`.
2. Once Rstudio is killed in that state, while it is in a project and in the hung state, there is no way of restarting the project unless I delete `.Rproj.user`
3. If I delete `.Rproj.user` I can start Rstudio again and go to the project. 
4. If I use `View()` it gets stuck again and I can only quit by killing  `rsession.exe` and then I am back to square one having to delete `.Rproj.user`
5. If I use New file the problem comes back!
6. I did not try deleting the the two folder and reinstalling as it was done in #1612 . Why would that matter? Wouldn't just deleting the profile folder be the same as deleting and reinstalling?
7. On Windows 10 I do not have this issue - Or maybe I haven't tested enough, but everything seems fine. 

```
> sessionInfo()
R version 3.4.2 (2017-09-28)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.4.2 tools_3.4.2    yaml_2.1.14
```

================================================================
Ok update:

- It could be that at work I have network drives, but I am using everything locally. In https://support.rstudio.com/hc/en-us/community/posts/115008640008-RStudio-1-1-very-slow-to-open-View-tabs-on-Windows-7 they suggest to set a HOME environment variable to a local one, but I don't even have that variable defined. I set it exactly as was recommended to the %USPERPROFILE% and that fixed the problem. But I don't know what setting it will break in other software.

**Comment by kevinushey** [comment]
Thanks for following up! We'll continue to investigate and try to get a fix soon.

**Comment by kevinushey** [comment]
I believe this issue should be resolved in v1.1.419. If you're still having trouble, can you please let us know in a new issue?
